seq_logger.js

```diff
@@ -50,18 +50,21 @@ class SeqLogger {
         });
     }
 
-    // Flush events queued at the time of the call, and wait for pending writes to complete
-    // regardless of configured batching/timers.
-    flush() {
+    /**
+     * Flush events queued at the time of the call, and wait for pending writes to complete regardless of configured batching/timers.
+     * @returns {Promise<boolean}
+     */
+    flush () {
```

Comment:
JSDoc can be seen in the intellisense:
![image](https://user-images.githubusercontent.com/2442596/125793902-2a049e65-8100-4a49-bcdc-c5258600c1e2.png)